verification/cbmc/proofs/aws_cryptosdk_hdr_write/Makefile

```diff
@@ -25,7 +25,7 @@ HARNESS_FILE = $(HARNESS_ENTRY).c
 # Values are chosen for performance. 
 # Increaseing either does not increase the coverage. 
 MAX_EDK_LIST_ITEMS ?= 1
-MAX_IV_LEN ?= 12
+MAX_IV_LEN ?= UINT8_MAX # Max value for uint8
```

Comment:
Could you add a comment why do we even need a bound for `MAX_IV_LEN`? I don't see it been used in loop unwinding, so I assume is related to arithmetic operations, but I couldn't find (quickly) any suggestion in the code for that, so a comment might help. @tegansb

Reply:
Added!